lib/scenic/adapters/postgres.rb

```diff
@@ -195,8 +196,9 @@ def drop_materialized_view(name)
       #   Scenic.database.refresh_materialized_view(:posts, concurrent: true)
       #
       # @return [void]
-      def refresh_materialized_view(name, concurrently: false)
+      def refresh_materialized_view(name, concurrently: false, cascade: false)
         raise_unless_materialized_views_supported
+        refresh_dependencies_for(name) if cascade
```

Comment:
We like to avoid inline conditionals, preferring the block form in most cases. Could you make that change here please?